src/UI/components/datahub/dataHubPanel.tsx

```diff
@@ -0,0 +1,83 @@
+import { Container } from '@mui/material';
+import React from 'react';
+import { Grid } from '@mui/material';
+import Image from 'next/image';
+import Link from 'next/link';
+
+function DataHubPanel(props: any) {
+  return (
+    <div>
+      <main>
+        <Container>
```

Comment:
You shouldn't need `div, main, Container` here - they're all on the dataHub page